apps/communications/contacts/test/unit/mock_multiple_select.js.html

```diff
@@ -0,0 +1,18 @@
+var MockMultipleSelectHTML =
```

Comment:
this could be loaded differently (by using XHR) in order not to duplicate the code